modules/qt/hm.nix

```
@@ -0,0 +1,64 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}: {
+  options.stylix.targets.qt = {
+    enable = config.lib.stylix.mkEnableTarget "QT" pkgs.stdenv.hostPlatform.isLinux;
+    iconThemeName = lib.mkOption {
+      description = "Default QT Icons";
+      type = lib.types.str;
+      default = "ePapirus-Dark";
+    };
+  };
+
+  config = lib.mkIf config.stylix.targets.qt.enable (let
+    cfg = config.stylix.targets.qt;
+    kvconfig = config.lib.stylix.colors {
+      template = ./kvconfig.mustache;
+      extension = ".kvconfig";
+    };
+    svg = config.lib.stylix.colors {
+      template = ./kvantum-svg.mustache;
+      extension = "svg";
+    };
+    kvantumPackage = pkgs.runCommandLocal "base16-kvantum" {} ''
+      directory="$out/share/Kvantum/Base16Kvantum"
+      mkdir --parents "$directory"
+      cat ${kvconfig} >>"$directory/Base16Kvantum.kvconfig"
+      cat ${svg} >>"$directory/Base16Kvantum.svg"
+    '';
+  in {
+    home.packages = with pkgs; [
+      qt5ct
```

```suggestion
      libsForQt5.qt5ct
```
Fixes `error: 'qt5ct' has been renamed to/replaced by 'libsForQt5.qt5ct'` in a recent nixos-unstable update